api-docs/api-reference/methods/token/post-authenticate-as-tenant-with-token-v2.0-tokens.rst

```diff
@@ -84,10 +78,6 @@ This table shows the body parameters for the request:
 |                          |                         |as a tenant.             |
 +--------------------------+-------------------------+-------------------------+
 
-
-
-
-
 **Example: Authenticate as tenant with token: XML request**
```

Comment:
In the other files in this PR, this type of heading had ^^^^^^ under it in the request and response sections. To be consistent between files, choose to either bold these with the ** or use the ^^^^ under the title. You might check other files to determine what's being used. (I think it might be to use bold with the ** before and after.)